dev/deploy-docs.sh

```diff
@@ -40,3 +40,10 @@ cd examples/doc
 make docs
 cd build/html
 aws s3 sync --delete --exclude ".*" --exclude "v/*" --acl public-read --cache-control "no-cache" ./ s3://flower.dev/docs/examples
+
+# Build and deploy Flower Android docs
+cd $ROOT
+cd src/kotlin
+./gradlew dokkaHtml
+cd build/dokka/html
+aws s3 sync --delete --exclude ".*" --exclude "v/*" --acl public-read --cache-control "no-cache" ./ s3://flower.dev/docs/android
```

Comment:
Similar to Swift lets separate the script so it can be used in a dedicated CI job